internal/connectors/engine/engine.go

```diff
@@ -176,26 +185,14 @@ func (e *engine) UninstallConnector(ctx context.Context, connectorID models.Conn
 		},
 		workflow.RunUninstallConnector,
 		workflow.UninstallConnector{
-			ConnectorID: connectorID,
+			ConnectorID:       connectorID,
+			DefaultWorkerName: defaultWorkerName,
 		},
 	)
 	if err != nil {
 		return err
 	}
 
-	// Wait for uninstallation to complete
-	if err := run.Get(ctx, nil); err != nil {
-		return err
-	}
```

Comment:
You could potentially still run this blocking call in a goroutine with a detached context if you want to delete the entry from the connectors table once the job is done.

Reply:
We don't need to do that, since the workflow is deleting the entry from the connectors table at the end. 

And it's a better idea to let the workflow do that instead of adding a new go routine with a detached context every time the payments service is launched. That way, we have all code at the same place !